README.md

```diff
@@ -15,16 +15,12 @@ See https://www.deltacast.tv for more video products.
 VideoViewer requires some dependencies to be installed on the system:
```

Comment:
Maybe we should also state that this repo uses git submodules. `git clone ... --recurse-submodules `

Reply:
Done